ansible/roles/lxc/defaults/main.yml

```diff
@@ -111,7 +111,7 @@ lxc__root_subgid_count: '{{ ansible_local.root_account.subgids[0]["count"]
 # :ref:`debops.lxd` role.
 lxc__net_deploy_state: '{{ "absent"
                            if ((ansible_virtualization_type in [ "lxc", "docker", "openvz" ] and
-                                ansible_virtualization_role == "aguest") or
+                                ansible_virtualization_role == "guest") or
```

Comment:
Yikes! That's a leftover from testing various conditions. Thanks for catching it.

Reply:
Don’t worry. You are welcome :)